package.json

```
@@ -24,13 +24,15 @@
     "graphql": "^15.0.0",
     "graphql-tools": "^5.0.0",
     "jsonwebtoken": "^8.5.1",
-    "merge-graphql-schemas": "^1.7.8"
+    "merge-graphql-schemas": "^1.7.8",
+    "uuid": "^8.1.0"
```

Remember setup fixed version to avoid issues.
```suggestion
    "merge-graphql-schemas": "1.7.8",
    "uuid": "8.1.0"
```